e2e/fixtures/orders/finalCareOrder.js

```diff
@@ -0,0 +1,20 @@
+module.exports =
+  {
+    type: 'Care order',
+    subtype: 'Final',
+    fullType: 'Final care order',
+    document: 'final_care_order.pdf',
+    dateOfIssue: {
+      day: '01',
+      month: '01',
+      year: '2020',
+    },
+    judgeAndLegalAdvisor: {
+      judgeTitle: 'Her Honour Judge',
+      judgeLastName: 'Judy',
+      judgeEmailAddress: 'test@test.com',
+      legalAdvisorName: 'Fred Frederickson',
+    },
+    closeCase: false,
```

Comment:
Are we testing in any flow closeCase true?

Reply:
We want to test closing the case through the close case event so we don't test the close case functionality from an order in the e2e, maybe that is an oversight on our part